app/controllers/api.controller.js

```diff
@@ -0,0 +1,9 @@
+const endpoints = require("../../endpoints.json")
+
+exports.getApi = (request, response) => {
+    return selectApi().then(() => {
```

Comment:
We don't need this line of code here, it seems like it isn't doing anything as you don't have a model and it's not being required in?